### PR TITLE
Documentation improvements

### PR DESCRIPTION
**Pull request by eyalbe4**
- [x] All [tests](https://github.com/jfrog/jfrog-vscode-extension#building-and-testing-the-sources) passed. If this feature is not already covered by the tests, I added new tests.
- [x] I used `npm run format` for formatting the code before submitting the pull request.
-----

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -124,30 +123,30 @@ The commands will do the following:
...
 
-You may provide basic auth credentials or access token as follows:
+You may set the connection details using the following environment variables. VS Code will read them after it is launched.
 
 -   `JFROG_IDE_URL` - JFrog URL
 -   `JFROG_IDE_USERNAME` - JFrog username
 -   `JFROG_IDE_PASSWORD` - JFrog password
 -   `JFROG_IDE_ACCESS_TOKEN` - JFrog access token
--   `JFROG_IDE_STORE_CONNECTION` - Set the value of this environment variable to **true**, if you'd like VS Code to store the connection details after reading them from the environment variables.
+-   `JFROG_IDE_STORE_CONNECTION` - Set the value of this environment variable to **true**, if you'd like VS Code to store the connection details after
```
Comment (attiasas) [inline README.md:149]: after connecting? after using them?

**Review by attiasas — CHANGES_REQUESTED** [review]

File: README.md [hunk 2]
```diff
@@ -124,30 +123,30 @@ The commands will do the following:
...
+Enter your JFrog platform connection details manually.
 
-**Note**: You can leave the platform URL empty, to enter the separate URLs for Artifactory and Xray.
+**Note**: You can leave the JFrog Platform URL empty, to enter the URLs of Artifactory and Xray separately.
 
-**Environment Variables**
+### Environment Variables
 
-You may provide basic auth credentials or access token as follows:
+You may set the connection details using the following environment variables. VS Code will read them after it is launched.
```
[the remark this answers is not in the document]
Comment (eyalbe4) [reply README.md:143]: Yes. Let me add them to the table of contents to remain consistent with the other items.

[the remark this answers is not in the document]
Comment (eyalbe4) [reply README.md:143]: On second thought, having those links in the table of contents will bypass the initial part of the **Connecting VS Code to Your JFrog Environment** section. I suggest we leave this as is.

File: README.md (hunk 1 above)
Comment (eyalbe4) [reply README.md:149]: Thanks!!
I think it should be "... after reading them from the environment variables."
I'll fix the sentence.

**Pull request merged**